.jenkins/common.groovy

```diff
@@ -55,18 +62,20 @@ def runCompileCommand(platform, project, jobName, boolean debug=false, boolean s
 def runTestCommand (platform, project) {
 
     String conformaceCPU = 'echo OpenVX 1.3 Conformance - CPU - NOT TESTED ON THIS PLATFORM'
-    String conformaceGPU = 'echo OpenVX 1.3 Conformance - GPU - NOT TESTED ON THIS PLATFORM'
+    String conformaceGPU_OpenCL = 'echo OpenVX 1.3 Conformance - GPU OpenCL - NOT TESTED ON THIS PLATFORM'
+    String conformaceGPU_HIP = 'echo OpenVX 1.3 Conformance - GPU HIP - NOT TESTED ON THIS PLATFORM'
     String moveFiles = ''
 
     if (platform.jenkinsLabel.contains('centos') || platform.jenkinsLabel.contains('ubuntu')) {
         conformaceCPU = 'AGO_DEFAULT_TARGET=CPU LD_LIBRARY_PATH=./lib ./bin/vx_test_conformance | tee OpenVX-CPU-Conformance-log.md'
-        conformaceGPU = 'AGO_DEFAULT_TARGET=GPU LD_LIBRARY_PATH=./lib ./bin/vx_test_conformance --filter=-HarrisCorners.*:-*.ReplicateNode:-*.ImageContainmentRelationship:-*.OnRandomAndNatural:-*.vxWeightedAverage:-vxCanny.*:-*.MapRandomRemap:*.* | tee OpenVX-GPU-Conformance-log.md'
+        conformaceGPU_OpenCL = 'AGO_DEFAULT_TARGET=GPU LD_LIBRARY_PATH=./lib ./bin/vx_test_conformance --filter=-HarrisCorners.*:-*.ReplicateNode:-*.ImageContainmentRelationship:-*.OnRandomAndNatural:-*.vxWeightedAverage:-vxCanny.*:-*.MapRandomRemap:*.* | tee OpenVX-GPU-OPENCL-Conformance-log.md'
+        conformaceGPU_HIP = 'AGO_DEFAULT_TARGET=GPU LD_LIBRARY_PATH=./lib ./bin/vx_test_conformance --filter=-.VirtualArray:-FastCorners.:-HarrisCorners.:-vxCanny.:-.ReplicateNode:-.ImageContainmentRelationship:-.GraphState:-.MapRandomRemap:-.OnRandomAndNatural:-.vxWeightedAverage:-Scale.GraphProcessing:-WarpPerspective.GraphProcessing:-Remap.GraphProcessing:-GaussianPyramid.GraphProcessing:-HalfScaleGaussian.GraphProcessing:. | tee OpenVX-GPU-HIP-Conformance-log.md'
```

Comment:
The HIP CTS filter is not correct and looks like that many stars (*) before dots (.) are missing in the filter.. I'll send you the correct filter.